index.js

```diff
@@ -167,6 +188,39 @@ let handleTrackName = async (channel) => {
     }
 }
 
+let handleQueue = async (channel) => {
+    try {
+        await printQueue(channel);
+    } catch (error) {
+        // Token expired
+        if(error?.response?.data?.error?.status === 401) {
+            await refreshAccessToken();
+            await printQueue(channel);
+        } else {
+            client.say(chatbotConfig.channel_name, `Seems like no music is playing right now`);
```

Comment:
Ideally most if not all messages that the chat can see should be configurable, this is a good default though.

Reply:
handleQueue is set up to function identically to the previously established handleTrackName method. 
![image](https://user-images.githubusercontent.com/22343767/201498005-ae9f1e0b-73fa-4a06-b6c2-9bee63fb4429.png)